crown-court-contribution/src/main/resources/schemas/maat-api/apiMaatCalculateContributionResponse.json

```diff
@@ -47,42 +43,15 @@
       "type": "string",
       "description": "Indicates what the contributions are based on (e.g. Means assessment)"
     },
-    "transferStatus": {
-      "type": "object",
-      "description": "The transfer status of the contribution record",
-      "existingJavaType": "uk.gov.justice.laa.crime.contribution.staticdata.enums.TransferStatus"
-    },
-    "dateUpliftApplied": {
-      "type": "string",
-      "format": "date-time",
-      "description": "The date that an income uplift was applied"
-    },
-    "dateUpliftRemoved": {
-      "type": "string",
-      "format": "date-time",
-      "description": "The date that an income uplift was removed"
-    },
     "createContributionOrder": {
       "type": "string",
       "description": "Indicates whether a contribution order should be created"
     },
-    "correspondenceId": {
-      "type": "integer",
-      "description": "The associated correspondence ID"
-    },
-    "active": {
-      "type": "string",
-      "description": "Indicates if this is a currently active contribution entry"
-    },
     "replacedDate": {
       "type": "string",
       "format": "date-time",
```

Comment:
Is there a reason for capturing the date fields in different formats - e.g. 'effectiveDate' as 'date', 'calcDate' as 'string', 'replaceDate' as 'date-time'?

Reply:
On the latest commit I can see we are using date-time for both the dates.